closed/autoconf/custom-spec.gmk.in

```diff
@@ -55,11 +55,13 @@ export GDK_HOME         := @OPENJ9_GDK_HOME@
 endif
 
 # feature enablement flags
-OPENJ9_ENABLE_CRIU_SUPPORT          := @OPENJ9_ENABLE_CRIU_SUPPORT@
-OPENJ9_ENABLE_DDR                   := @OPENJ9_ENABLE_DDR@
-OPENJ9_ENABLE_DEMOS                 := @OPENJ9_ENABLE_DEMOS@
-OPENJ9_ENABLE_JITSERVER             := @OPENJ9_ENABLE_JITSERVER@
-OPENJ9_ENABLE_OPENJDK_METHODHANDLES := @OPENJ9_ENABLE_OPENJDK_METHODHANDLES@
```

Comment:
This should not have been removed (`--enable-openjdk-methodhandles` is still a valid configuration option).

Reply:
Fixed.